
From 997bc0d12fa3dbbf21582d41f657a6caff25c583 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 6 Sep 2021 15:46:25 +0200
Subject: [PATCH] Trigger cleanup faster

Call cleanup of the database faster to allow smaller databases.
---
 monitor.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/monitor.py
+++ b/monitor.py
@@ -343,7 +343,7 @@ def main():
     # it's recommended not to touch this, unless you know really well what you're doing
     # filling all available space in /var/lib (tmpfs) will probably break your router
     hard_limit = int(uci_get('pakon.monitor.database_limit') or 3000000)
-    run_check = everyN(100000)
+    run_check = everyN(1000)
     while True:
         try:
             line = data_source.get_message()
